atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/CellToSweep.java

```diff
@@ -15,22 +15,32 @@
  */
 package com.palantir.atlasdb.sweep;
 
+import java.util.Optional;
```

Comment:
Per an internal (team-atlasdb, July 5) chat discussion, it seems we want to start using Java 8 Optionals where possible. Let me know if you see this causing any problems though, and I'm happy to switch back.